axelor-sale/src/main/resources/views/SaleOrderLine.xml

```diff
@@ -76,6 +80,10 @@
       onClick="save,action-sale-order-line-view-regenerate-line"
       if="__config__.app.getApp('sale')?.getEnableConfigurator()" readonlyIf="configurator == null"
       icon="fa-pen-to-square"/>
+    <button name="configuratorDuplicateBtn"
+      onClick="action-sale-order-line-method-configurator-duplicate"
```

Comment:
it seems that a save is missing here ? else when we call `setReload(true)` in the controller, it will remove every unsaved change in the form view.